.eslintrc

```diff
@@ -27,7 +27,8 @@
     // prettier config
     "prettier/prettier": "error",
     "eqeqeq": ["error", "smart"],
-    "comma-dangle": ["error", "never"]
+    "comma-dangle": ["error", "never"],
+    "react/jsx-handler-names": 1
```

Comment:
Can you please explain why you added this here ?

Reply:
This rule enforces the use of the correct prefix for event handling events. IMO it's too strict.